backend/src/models/request.py

```diff
@@ -6,4 +6,4 @@ class UserMessage(BaseModel):
 
 class ChatRequest(BaseModel):
     message: list[UserMessage]
-    stream: bool = True
\ No newline at end of file
+    stream: bool = False
```

Comment:
Has this been coordinated with @SOCRAMBLLE from the frontend team? We might aim to go for streaming in the MVP - only if it is not too cumbersome.